packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -46,6 +46,8 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
           selectedCount={props.selected.length}
           onClearAll={() => {
             props.selectedStateSetter([]);
+            props.setSearchValue("");
+            props.handleSearchChange("");
```

Comment:
hey sorry I wasn't part of the slack thread on this but I don't agree with these changes

the clear all button should be able to be used based on the context provided by the search term, but it shouldn't be able to change it

the select/clear all is more about the selected state and what items we have, not the current search term